hub/core/storage/local.py

```diff
@@ -0,0 +1,28 @@
+from hub.core.storage.provider_mapper import ProviderMapper
+import fsspec
+
+
+class LocalProvider(ProviderMapper):
+    """
+    Provider class for using the local filesystem.
+    """
+
+    def __init__(self, root: str):
+        """
+        Initializes the LocalProvider.
```

Comment:
I think [google docstrings](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) require that the first statement be inline with the quotes.

```
"""Like this

...
"""
```